dp/generate_climos.py

```diff
@@ -739,13 +753,18 @@ def update_generate_climos_history(netCDF_file, time_cdo_format, position=0):
         ...
 
     """
+    # stdin input passed through terminal
```

Comment:
Arguments on the command line are not standard input. Standard input is a file stream that every process gets (whether or not it is used, which it isn't in this case). Command line arguments are a different type of input.

I'd revise this comment since it's confusing (and incorrect).